cypress/plugins/index.js

```diff
@@ -253,6 +242,33 @@ function deleteCollectionRecursively(collection) {
   });
 }
 
+/**
+ * Stores necessary data for tests to consume from prod Firebase. Called
+ * once at the start of all tests. We
+ * add a dummy field at the top level of the document so that Firestore will
+ * deign to list this document later
+ * (https://stackoverflow.com/questions/47043651/this-document-does-not-exist-and-will-not-appear-in-queries-or-snapshots-but-id)
+ * @return {Promise<null>} Promise that completes when retrieval is done
+ */
+function storeTestFirestoreData() {
+  const prodApp = firebase.initializeApp(firebaseConfigProd, 'prodapp');
+  const readPromises = [];
+  for (const disaster of ['2017-harvey', '2018-michael']) {
+    const documentPath = 'disaster-metadata/' + disaster;
+    const prodDisasterDoc = prodApp.firestore().doc(documentPath);
+    readPromises.push(prodDisasterDoc.get().then((result) => {
+      result.data().dummy = true;
+      return {disaster, data: result.data()};
+    }));
+  }
+  return Promise.all(readPromises)
+      .then(async (result) => {
```

Comment:
Sorry, can you explain what async does here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function

It allows us to put an await in here and then return the result of the promise without jumping through hoops.